### ISSUE TITLE
Windows Vista/Server 2008 Compatibility

### ISSUE THREAD

**Issue by EFIShell0** [manatails/uefiseven#39]
Does it work on Windows Vista/Server 2008?  You know, the same situation exists with him.  Does it work in RTM/SP1/SP2?

**Comment by Gelip** [comment]
@EFIShell0
Yes, works - tested winload.exe x86 5472.5 (also works with XP32)
